### ISSUE TITLE
web ui internal server error 500 displaying a maintenance request

### ISSUE THREAD

**Issue by andreasstieger** [openSUSE/open-build-service#1564]
A particular maintenance request throws a internal server error:
https://build.opensuse.org/request/show/360269

This works:
osc request show 360269
osc request show 360269 -d

maintenance_incident: home:AndreasStieger:branches:OBS_Maintained:fate/fate.openSUSE_Leap_42.1_Update -> openSUSE:Maintenance (release in openSUSE:Leap:42.1:Update)

**Comment by hennevogel** [comment]
Obviously you can set by_project to whatever you want (in this case a package)

```
<review state="new" by_project="FATE/FATE-stable">
    <comment>Switching from the SLE codestream package to FATE/FATE-stable for boo#966821. Is that okay for you?</comment>
  </review>
```

**Comment by andreasstieger** [comment]
Ah yes I entered it in the web ui. Adjusted description. Probably low prio, can be worked around.

**Comment by adrianschroeter** [comment]
urgs, the lacking validation is a regression since api move ... fixing ...

**Comment by adrianschroeter** [comment]
fixed two weeks ago with 5f73b71bdcba50a991eb3c5e8c4b5ccf428c449e
